src/sindri/sindri.py

```diff
@@ -1,3 +1,26 @@
+"""
+## Usage
+
+```python
+# Download some sample data
+# git clone https://github.com/Sindri-Labs/sindri-resources.git
+
+# pip install sindri
+from sindri.sindri import Sindri
+
+# Initialize
+sindri = Sindri("<YOUR_API_KEY>", verbose_level=2)
```

Comment:
Maybe leave out `verbose_level`? Is that required to get the job done?

Reply:
This docstring is displayed on the generated docs page as a small quickstart. The default verbose_level is zero (print nothing), so I think we should include it here so something is printed for a quickstart-follower. What do you think?

Reply:
I'd like to know where, in a public document, I can see that docstring. Thanks!

Reply:
See slack conversation for more details